docs/administration/projects/node-execution/powershell.md

```diff
@@ -186,6 +192,13 @@ To enable the CredSSP:
 winrm set winrm/config/service/auth @{CredSSP="true"}
 ```
 
+To enable unencrypted access run this command on the Rundeck server and the remote nodes:
```

Comment:
Why would someone want to do this?  Running "unencrypted" seems like a bad idea, so let's put context around the use case.  That way it doesn't seem like we're encouraging lower security.